contracts/cw3-fixed-multisig/src/contract.rs

```diff
@@ -394,9 +394,11 @@ fn map_proposal(
     })
 }
 
-fn query_vote(deps: Deps, proposal_id: u64, voter: HumanAddr) -> StdResult<VoteResponse> {
-    let voter_raw = deps.api.canonical_address(&voter)?;
-    let ballot = BALLOTS.may_load(deps.storage, (proposal_id.into(), &voter_raw))?;
+fn query_vote(deps: Deps, proposal_id: u64, voter: String) -> StdResult<VoteResponse> {
+    let ballot = BALLOTS.may_load(
+        deps.storage,
+        (proposal_id.into(), AddrRef::unchecked(&voter)),
```

Comment:
This is an interesting diff. Before the user input was checked and only if it was valid, the query to the data store was made. I think we should continue to do that. The other option is to use String/str in the map.

But fake-casting to `Addr` seems to defeat the whole purpose if the strict type.